tests/encodehelp.h

```diff
@@ -220,9 +226,15 @@ void setEncoderParameters(VideoParamsCommon * encVideoParams)
     encVideoParams->rcParams.initQP = initQp;
     encVideoParams->rcMode = rcMode;
     encVideoParams->numRefFrames = numRefFrames;
+    encVideoParams->bitDepth = bitDepth;
     //encVideoParams->rcParams.minQP = 1;
 
-    //encVideoParams->profile = VAProfileH264Main;
+    if(bitDepth == 10 && encVideoParams->profile == VAProfileHEVCMain)
+    {
+        encVideoParams->profile = VAProfileHEVCMain10;
```

Comment:
we are plan to remove all VA things from interface, please do not use encVideoParams->profile. since you have bit depth, why you need this?